compiler/il/OMRNode.cpp

```diff
@@ -3360,6 +3360,46 @@ OMR::Node::getOwningMethod()
 
 
 
+TR_OpaqueMethodBlock*  
+OMR::Node::getOwningMethod(TR::Node *node, TR::Compilation *comp, bool isIProfilingEnabled)
+   {
+   if (isIProfilingEnabled) 
+      return 0;  
+  
+   TR_ByteCodeInfo bcInfo = node->getByteCodeInfo(); 
```

Comment:
Rather than casing the code here it would probably make sense for the node implementation to just extract the node bcInfo and delegate to the BCInfo based version - there is no need for this kind of duplication here...